docs/community/com-db/com-db-manage.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Manage Database
```

Comment:
I'm a little confused at what this page is trying to be, especially when compared to cloud: https://docs.featurebase.com/docs/cloud/cloud-databases/cloud-db-manage/ . It has a low level fact about DBs and then goes to things you can do with them

Reply:
Well there's not much information on community dbs in the help, other than in "old-size-featurebase-database.md" which is actually more about pre-installation requirements than databases. Or put another way, the user wouldn't do this after FeatureBase is installed because it involves assessing RAM and HD/SSD size.

Thus the page is lighter on content and is more a short summary to explain the basics of the database, and that it doesnt need to be setup (like Cloud).

Does that make sense? Can you point me to information that could go in here?